src/PositionManager/PositionManager.h

```diff
@@ -54,12 +57,14 @@ private slots:
     void gcsPositionChanged(QGeoCoordinate gcsPosition);
     void gcsHeadingChanged(qreal gcsHeading);
     void positionInfoUpdated(QGeoPositionInfo update);
+    void gcsPositionHorizontalAccuracyChanged();
 
 private:
     int                 _updateInterval =   0;
     QGeoPositionInfo    _geoPositionInfo;
     QGeoCoordinate      _gcsPosition;
     qreal               _gcsHeading =       qQNaN();
+    qreal               _gcsPositionHorizontalAccuracy = std::numeric_limits<qreal>::infinity();
```

Comment:
Is there a benefit to using infinity here? In the rest of QGC NaN is used to indicate values being unavailable. To me this should follow that patter and this should be NaN for unavailable.

Reply:
I think there is @DonLakeFlyer , consider:
```
    auto isGoodEnoughAccuracyToDoSomethingUseful = [](double accuracy) {
        if (accuracy > 100) {
            std::cout << "accurancy of " << accuracy << " is poor" << std::endl
                      << std::flush;

        } else {
            std::cout << "accurancy of " << accuracy << " is sufficent" << std::endl
                      << std::flush;
        }
    };
    isGoodEnoughAccuracyToDoSomethingUseful(std::numeric_limits<double>::infinity());
    isGoodEnoughAccuracyToDoSomethingUseful(qQNaN());
```
the output is:
```
accurancy of inf is poor
accurancy of nan is sufficent
```
infinite accuracy means infinitely poor accuracy maintaining fidelity to the greater than/less than operators. NaN will not do that - the result of comparison with any other double is always false.